components/ConnectWalletButton.tsx

```diff
@@ -163,7 +162,17 @@ const ConnectWalletButton = (props) => {
                       </button>
                     </Menu.Item>
                   ))}
-
+                  <Menu.Item key={'devnet'}>
```

Comment:
Why change the location of the toggle inside the menu?
Unrelated to current scope

Reply:
That was a fix that Sebastian wanted. Otherwise, devnet toggle is only available when a wallet is connected

Reply:
Ok :+1: